tests/foundry/contracts.k.check.stripped

```diff
@@ -0,0 +1,7360 @@
+requires "foundry.md"
+
+module ACCOUNTPARAMSTEST-CONTRACT
+    imports public FOUNDRY
+    
+    syntax Contract ::= AccountParamsTestContract
+    
+    syntax AccountParamsTestContract ::= "AccountParamsTest" [symbol(), klabel(contract_AccountParamsTest)]
+    
+      
+    
+    syntax Field ::= AccountParamsTestField
+    
+    syntax AccountParamsTestField ::= "IS_TEST" [symbol(), klabel(field_AccountParamsTest_IS_TEST)]
+    
+    syntax AccountParamsTestField ::= "_failed" [symbol(), klabel(field_AccountParamsTest__failed)]
+    
```

Comment:
Is this file supposed to be added? I think it's just for comparing output with? My guess is it got added accidentally?

Maybe it should be added to `.gitignore` to make sure this doesn't happen again?